app/services/travel_to_work_area_and_london_borough_service.rb

```diff
@@ -0,0 +1,19 @@
+class TravelToWorkAreaAndLondonBoroughService
+  def self.add_travel_to_work_area_and_london_borough(site:)
+    url = URI("#{Settings.mapit_url}/point/4326/#{site.longitude},#{site.latitude}?type=TTW,LBO&api_key=#{Settings.mapit_api_key}")
+    response = Net::HTTP.get(url)
+    json = JSON.parse(response)
+
+    ttw_area = json.dig(json.keys.first, "name")
+
+    london_borough = if ttw_area == "London"
+                       json.dig(json.keys.second, "name")
+                     end
+
+    site.update_column("travel_to_work_area", ttw_area)
+    site.update_column("london_borough", london_borough.gsub(/ Borough Council| City Council| Corporation/, ""))
```

Comment:
I see the 'Borough' string is being removed - do we need to backfill this change on prod?

Reply:
Also this substitution does not work on "Royal London Borough of ..."

Might need to extract and unit test?

Reply:
I had a quick test on Mapit and it doesn't actually return Royal London Borough for any of them (Kingston, Kensington etc.)

![image](https://user-images.githubusercontent.com/42515961/108213697-3c0bc680-7127-11eb-8466-2ea669331842.png)

![image](https://user-images.githubusercontent.com/42515961/108213816-580f6800-7127-11eb-9a15-a6f18fa86438.png)

![image](https://user-images.githubusercontent.com/42515961/108213954-7aa18100-7127-11eb-93ff-0160b58bd403.png)

Reply:
I just backfilled prod 
![image](https://user-images.githubusercontent.com/42515961/108214619-45496300-7128-11eb-8e00-1617d68d811b.png)